src/main/java/io/prowave/chargify/webhook/bean/CustomerUpdate.java

```diff
@@ -20,27 +20,16 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class CustomerUpdate {
+public class CustomerUpdate extends Payload{
 
-	@JsonProperty("event_id")
-	private Long eventId;
-	private Site site;
 	private Customer customer;
 
-	public Long getEventId() {
-		return eventId;
+	public Subscription getSubscription() {
+		return null;
```

Comment:
CustomerUpdate was an orphan, all other classes extend Payload. I needed to have a common subclass for my code, so I made CustomerUpdate a subclass of Payload.

This meant it inherited Subscription, and its getters and setters. I overrode them so they could not be used, as CustomerUpdate which does not have Subscription.

A better solution would be to remove Subscription from Payload. Then subclass Payload with SubscriptionPayload, and move the current Payload classes to extend SubscriptionPayload. But that would break code that relied on Payload having a Subscription, and I didnt want to do that.